ansible/validate.yml

```
@@ -77,6 +77,7 @@
           - cluster_home_volume is defined
           - cluster_compute_groups is defined
         fail_msg: "One or more expected variables are missing: is OpenTofu inventory template up to date?"
+      when: not appliances_caas_skip_validate_vars
```

I'd rather spell this like this (and default it to `true` in common), but don't bother if it means more work now.
```suggestion
      when: appliances_validate_tofu_vars | bool
```
(using bool for ease of CLI extravars overriding).